app/views/document_change_requests/new.html.erb

```diff
@@ -0,0 +1,60 @@
+<% add_parent_breadcrumb_link "Home", planning_applications_path %>
+<% add_parent_breadcrumb_link "Application", validate_documents_form_planning_application_path(@planning_application) %>
+<% add_parent_breadcrumb_link "New change request", new_planning_application_change_request_path(@planning_application) %>
+
+<% content_for :title, "Request replacement documents" %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h2 class="govuk-heading-l govuk-!-padding-top-6">
+      Request replacement documents
+    </h2>
+    <p class="govuk-body govuk-!-padding-bottom-3">
+      The following documents have been marked as invalid.
+    </p>
+    <%= form_with model: [@planning_application, @document_change_request], local: true, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |form| %>
+      <% if @document_change_request.errors.any? %>
```

Comment:
We'll never have errors on this object (500 instead), so we can remove this HTML